landing-strip/core/src/main/java/com/novoda/landingstrip/Coordinates.java

```diff
@@ -0,0 +1,6 @@
+package com.novoda.landingstrip;
+
+public interface Coordinates {
```

Comment:
it's because of the api, the clients shouldn't be able to modify the properties, only the lib can do that